template/golang-middleware/Dockerfile

```diff
@@ -12,14 +12,21 @@ RUN mkdir -p /go/src/handler
 WORKDIR /go/src/handler
 COPY . .
 
+RUN $(cat function/GO_REPLACE.txt >> ./go.mod || exit 0) && cat go.mod
```

Comment:
catting the go,mod at the end of this will provide useful feedback that the replace worked, but it is adding verbosity to build output

Reply:
Sure, happy to remove.